kustomization/components/operator-prometheus/subscription.yml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: my-prometheus
+  namespace: operators
+spec:
+  channel: beta
+  name: prometheus
+  source: operatorhubio-catalog
+  sourceNamespace: olm
```

Comment:
Not being familiar with Prometheus or the operator framework, these lines could use some commenting to explain what they do and how to customize them (if possible, necessary), or perhaps docs on the operator framework component itself.

Reply:
This is taken verbatim from the install instructions on https://operatorhub.io/operator/prometheus

The `operatorhubio-catalog` is part of the olm installation, which lives in the `olm` namespace.